replay-test/utils.ts

```diff
@@ -0,0 +1,26 @@
+export function assert(v: any, why = ""): asserts v {
```

Comment:
Why do we need this custom `assert` function? Can we use the built-in node `assert` function instead?

Reply:
That would be fine here, I copied this from the backend repo where we do want more control over the behavior of assert().